rest_framework-stubs/viewsets.pyi

```diff
@@ -12,11 +12,18 @@ def _is_extra_action(attr: Any) -> bool: ...
 _ViewFunc = Callable[..., HttpResponseBase]
 
 class ViewSetMixin(object):
+    # Classvars assigned in as_view()
     name: Optional[str]
     description: Optional[str]
     suffix: Optional[str]
     detail: bool
     basename: str
+    # Instance attributes assigned in view wrapper
+    action_map: Dict[str, str]
+    args: List[Any]
```

Comment:
Its't it `Tuple[Any, ...]`?

Reply:
Oh yes, you are right. Finally found the time to fix this.